Makefile

```diff
@@ -2,37 +2,37 @@ testjs: ## Clean and Make js tests
 	yarn test
 
 testpy: ## Clean and Make py tests
-	python3.7 -m pytest -v jupyterfs/tests --cov=jupyterfs --cov-branch --junitxml=python --cov-report=xml
+	python3.7 -m pytest -v jupyterfs/tests --cov=jupyterfs --cov-branch --junitxml=python_junit.xml --cov-report=xml
 
 test: ## run all tests
-	testpy
-	testjs
+	make testpy
+	make testjs
 
 lintjs: ## run linter
 	yarn lint
 
 lintpy: ## run linter
-	flake8 jupyterfs setup.py
+	python3.7 -m flake8 jupyterfs setup.py
 
 lint: ## run linter
-	lintpy
-	lintjs
+	make lintpy
+	make lintjs
```

Comment:
Wait. I thought that calling one target from another target in a makefile always worked. Are there instances where it won't?